### PR TITLE
boolean default val

### PR DESCRIPTION
**Pull request by paribaker**
fix for [issue5](https://github.com/thinknimble/tn-models/issues/5#issue-1116768154) [ issue4](https://github.com/thinknimble/tn-models/issues/4#issue-1032754404)

Moved order of defaultVal, this should not affect current implementations just allow users to set a default value, tests added to confirm.
Updated readme to remove default val from charfield as an option
@whusterj

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -78,12 +84,15 @@ describe('Model', function () {
         lastName: 'Hackerman',
         bestFriend: null,
         allFriends: null,
+        isCool: false,
+        isHawt: false,
+        isGlam: false,
```
Comment (whusterj) [inline test/test.js:89]: 🥇

File: package.json [hunk 2]
```diff
@@ -29,13 +29,16 @@
...
     "@babel/preset-env": "^7.9.5",
     "@babel/register": "^7.9.0",
+    "@babel/runtime": "^7.16.7",
     "mocha": "^7.1.1"
   },
   "babel": {
     "plugins": [
-      "@babel/plugin-proposal-class-properties"
+      "@babel/plugin-proposal-class-properties",
+      "@babel/transform-runtime"
```
Comment (whusterj) [inline package.json:41]: @paribaker - Why is this new transform needed?

**Review by whusterj — APPROVED** [review]
Looks good! I am just curious about the new Babel transform, since that seems like it may be outside the scope of the main change.

There are also a lot of code format changes. So I also want to confirm that you are using prettier, so those changes are in line with our current standards.

File: package.json (hunk 2 above)
Comment (paribaker) [reply package.json:41]: I came across this error when trying to run tests after a couple of posts around the topic I managed to get it to work from the babel docs

https://stackoverflow.com/questions/57219314/error-while-trying-to-run-scripts-regeneratorruntime-is-not-defined

**Pull request merged**